modules/ingest-geoip/src/test/resources/plugin-security.policy

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+grant {
```

Comment:
Why is this needed?

Reply:
this was needed to make tests work with painless, I removed it now as we don't use update by query anymore